From ef874a272e78c16e6d96ef1e3f9f9f81c24b54b3 Mon Sep 17 00:00:00 2001
From: Sasha Levin <sashal@kernel.org>
Date: Tue, 10 Jan 2023 13:49:23 +0100
Subject: s390/con3270: move condev definition

From: Heiko Carstens <hca@linux.ibm.com>

[ Upstream commit 9cab4f7d98eee90b762ffbc87055c9499d51a635 ]

Fix this for allmodconfig:

drivers/s390/char/con3270.c:43:24: error: 'condev' defined but not used [-Werror=unused-variable]
 static struct tty3270 *condev;
                        ^~~~~~

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Fixes: c17fe081ac1f ("s390/3270: unify con3270 + tty3270")
Signed-off-by: Heiko Carstens <hca@linux.ibm.com>
Signed-off-by: Sasha Levin <sashal@kernel.org>
---
 drivers/s390/char/tty3270.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/s390/char/tty3270.c b/drivers/s390/char/tty3270.c
index eee2030b86b3a..583f2aecbed3c 100644
--- a/drivers/s390/char/tty3270.c
+++ b/drivers/s390/char/tty3270.c
@@ -38,7 +38,6 @@
 
 static struct tty_driver *tty3270_driver;
 static int tty3270_max_index;
-static struct tty3270 *condev;
 static struct raw3270_fn tty3270_fn;
 
 struct tty3270_cell {
@@ -1910,6 +1909,9 @@ static void __exit tty3270_exit(void)
 }
 
 #if IS_ENABLED(CONFIG_TN3270_CONSOLE)
+
+static struct tty3270 *condev;
+
 static void
 con3270_write(struct console *co, const char *str, unsigned int count)
 {
-- 
2.53.0

